tools/DevCheck/DevCheck.ps1

```diff
@@ -166,7 +166,7 @@ Param(
 
     [Switch]$NoInteractive=$false,
 
-    [String]$NugetExe='.user\nuget.exe',
+    [String]$NugetExe="$PSScriptRoot\..\..\.user\nuget.exe",
```

Comment:
The relative pathname's kinda icky, and unnecessary. DevCheck has functions

* Get-ProjectRoot
* Get-UserPath

among others. Make this

```
[String]$NugetExe=[IO.Path]::GetFullPath(Join-Path Get-UserPath "nuget.exe"),
```

Or is the GetFullPath unnecessary given `Get-UserPath` = `Join-Path Get-ProjectRoot .user` and `Get-ProjectRoot` = `return (Get-Item $PSScriptRoot ).parent.parent.FullName` ?

Reply:
NOTE: `Get-UserPath` ensures the `.user` directory exists before returning, creating if necessary. Is that a problem for a default parameter to DevCheck?

If so then make it

```
[String]$NugetExe=[IO.Path]::GetFullPath(Join-Path Get-ProjectRoot ".user\nuget.exe"),
```

Reply:
The main complication here is that the `Get-ProjectRoot` is not defined yet in the script at the moment the default value for the parameter is processes, so we can't simply call it there. I used the relative path for not overcomplicating it.